py/kubeflow/kubeflow/cd/kaniko_builder.py

```diff
@@ -9,10 +9,12 @@ def __init__(self, name=None, namespace=None, bucket=None,
         super().__init__(name=name, namespace=namespace, bucket=bucket,
                          test_target_name=test_target_name, **kwargs)
 
-    def build(self, dockerfile, context, destination):
+    def build(self, dockerfile, context, destination,
+              second_dockerfile=None, second_destination=None,
```

Comment:
I think these args aren't used anywhere

Reply:
They are used [here[(https://github.com/kubeflow/kubeflow/pull/5627/files#diff-d863e305a849abf34adc003f1657663d9261aa4c3fab0849dc71dfb761b3475d). Like I mentioned above I've added these changes to this PR to make the huge amount of rebasing I will need to do later on a little easier.

Reply:
As @PatrickXYS mentioned above, could you edit the description to reflect what changes were made to assist the rebases of future PRs? 

This will help us keep track of the progress and chain between these PRs

Reply:
Sure thing, I had actually not intended the `second_*` being in here (which is why I didn't add the lines below), classic copy/paste error haha. I've removed the `second_dockerfile=None, second_destination=None` from this PR.